src/sensors/TallyCounterI2C.h

```diff
@@ -0,0 +1,90 @@
+/*
+ *TallyCounterI2C.h
+ *This file is part of the EnviroDIY modular sensors library for Arduino
+ *
+ *Initial library developement done by Sara Damiano (sdamiano@stroudcenter.org).
+ *
+ *This file is for an external Event counter, used to measure windspeed or rainfall
+ *
+ *Documentation for the sensor can be found at:
+ * https://github.com/NorthernWidget-Skunkworks/Project-Tally​
+ * https://github.com/NorthernWidget-Skunkworks/Tally_Library/tree/Dev_I2C
+ *
+ * For Wind Speed:
+ *  Accuracy and resolution are dependent on the sensor used:
+ *  Inspeed WS2R Version II Reed Switch Anemometer
+ *    https://www.store.inspeed.com/Inspeed-Version-II-Reed-Switch-Anemometer-Sensor-Only-WS2R.htm
+ *
+ * Assume sensor is immediately stable
+*/
+
+// Header Guards
+#ifndef TallyCounterI2C_h
+#define TallyCounterI2C_h
+
+// Debugging Statement
+// #define MS_TALLYCOUNTERI2C_DEBUG
+
+#ifdef MS_TALLYCOUNTERI2C_DEBUG
+#define MS_DEBUGGING_STD "TallyCounterI2C"
+#endif
+
+// Included Dependencies
+#include "ModSensorDebugger.h"
+#undef MS_DEBUGGING_STD
+#include "VariableBase.h"
+#include "SensorBase.h"
+#include <Tally_I2C.h>
+
+// Sensor Specific Defines
+#define TALLY_NUM_VARIABLES 1
+#define TALLY_WARM_UP_TIME_MS 0
+#define TALLY_STABILIZATION_TIME_MS 0
+#define TALLY_MEASUREMENT_TIME_MS 0
+
+#define TALLY_EVENTS_RESOLUTION 0
+#define TALLY_EVENTS_VAR_NUM 0
+
+// The main class for the external event counter
+class TallyCounterI2C : public Sensor
+{
+public:
+    // The constructor - all arguments are optional
+    // Address of I2C device is 0x08 by default
+    // Depth of rain per tip event in mm is 0.2mm by default
```

Comment:
fix comments

Reply:
Done by you with https://github.com/EnviroDIY/ModularSensors/pull/269/commits/5a972b5f40972eec05b4b278ac9264f63ced1acc